lib/compile.js

```diff
@@ -130,7 +131,7 @@ var Compiler = (function () {
                     msgid = msgid.replace( unconvertedEntityPattern, convertedEntity );
                 }
 
-                if ( removeWhiteSpaces ) {
+                if ( removeWhiteSpaces && !isDBString ) {
```

Comment:
I'd suggest keeping removeWhiteSpaces here and make isDBString a constituent of the value defined in 
https://github.com/udemy/angular-gettext-tools/blob/5a7eb6ad86667970760d9150da42338fa6862412/lib/compile.js#L107